### PR TITLE
[13] - Test save wallet in cache

### LINKED ISSUE #13: Test saveWalletInCache

### PR DESCRIPTION
**Pull request by IvanSolaDelgado**
**Issue**
- Resolves #13 

**Solution**
- I have tested the case in which the cache is full and the case in which the cache is not full and we save the wallet.

### CODE REVIEW

File: tests/app/Application/DataSources/WalletDataSourceTest.php [hunk 1]
```diff
@@ -22,6 +23,33 @@ protected function setUp(): void
         });
     }
 
+    /**
+     * @test
+     */
+    public function whenCacheIsNotFullSavesWalletInCache()
```
Comment (AsierAlba10) [inline tests/app/Application/DataSources/WalletDataSourceTest.php:29]: saveWalletWhenCacheIsNotFull??

To choose test names we must first think about what the test will do and then why or when.
<action><when/why>

saveWallet><WhenCacheIsNotFull

File: tests/app/Application/DataSources/WalletDataSourceTest.php [hunk 2]
```diff
@@ -22,6 +23,33 @@ protected function setUp(): void
...
+            ->once()
+            ->with('wallet_0', Mockery::type('array'));
+
+        $this->assertEquals('wallet_0', $walletDataSource->saveWalletInCache());
+    }
+
+    /**
+     * @test
+     */
+    public function whenCacheIsFullReturnsNull()
```
Comment (AsierAlba10) [inline tests/app/Application/DataSources/WalletDataSourceTest.php:44]: returnsNullWhenCacheIsFull?

**Review by AsierAlba10 — APPROVED** [review]
Goood, change the test names and ok to merge

File: tests/app/Application/DataSources/WalletDataSourceTest.php (hunk 1 above)
Comment (IvanSolaDelgado) [reply tests/app/Application/DataSources/WalletDataSourceTest.php:29]: Top!! Thanks ⭐

**Pull request merged**